nixos/modules/services/networking/unifi.nix

```diff
@@ -148,7 +148,7 @@ in
 
     systemd.tmpfiles.rules = [
       "e '${stateDir}' 0700 unifi - - -"
```

Comment:
Should `e` actually be a `Z`?

Reply:
I think this would fail, because `${stateDir}/bin` for example is a bind mount from nix store. I will try in a moment and report back.
It would probably also not resolve the issue with `${stateDir}/data` because it also only acts on existing files according to the manpage.

Reply:
Yes as expected:
```
systemd-tmpfiles[499]: fchmod() of /var/lib/unifi/lib failed: Read-only file system
``

Reply:
Ah I see. Yeah I missed that part. Not familiar enough with the module. What you're saying makes sense to me.